tests/test_photometryModel.py

```diff
@@ -322,10 +322,12 @@ def test_checkPositiveOnBBox(self):
         self.assertFalse(self.model.checkPositiveOnBBox(self.ccdImageList[0]))
 
     def test_validate(self):
-        self.assertTrue(self.model.validate(self.ccdImageList))
-        # Make the model go negative
+        # Model has 20 parameters (2 visits, 10 params each)
```

Comment:
Please add a comment like in the astrometryModel test that explains why the assertion with 1 dof should be True but the one with 0 dof should be False.